script.js

```diff
@@ -1,4 +1,198 @@
+//Adding username
+var userName = "";
+
+//Wins and total games played
+var numWins = 0;
+var numTotalGames = 0;
+
 var main = function (input) {
-  var myOutputValue = 'hello world';
-  return myOutputValue;
+  //Thanks to Tinaes for teaching array
+  console.log(userName + ": " + input);
+  var moves = ["scissors", "paper", "stone"];
+
+  var compMove = Math.floor(Math.random() * moves.length); //Computer random moves
+
+  var rMoves = ["reversed scissors", "reversed paper", "reversed stone"];
+
+  var rCompMove = Math.floor(Math.random() * rMoves.length); //RComputer random moves
```

Comment:
good use of arrays. as compMove and rCompMove has yet to be scissors paper or stone it can be better named as a number or index